### DISCUSSION TITLE
Welcome to Winter Discussions!

### DISCUSSION THREAD

**Discussion by LukeTowers** [wintercms/winter#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by impactfactory** [comment]
Im just very thankful that you keep OC/WN Open Source - thank you so much. As a generalist I love to be able to build this fast, modern websites very much. 

If there is ever interest in that: I developed my own blog plugin as I found the Rainlab Blog Plugin too hard to design. Im not sure if I could make it ready for a marketplace, but it already is multi-language (de and en in both front and backend), searchable, sluggable, listv iew, detail view, category view, tags view, rights management, tag cloud, webp, froala based with snippets, some editorial status overview in the post list and focused cropping for keyvisuals and provides the breadcrumb and blogposting rich snippets, also all other meta tags needed. all with 90's on Lighthouse. :)

https://github.com/impactfactory/blog-plugin/blob/main/README.md
